source/slang/slang-emit-c-like.h

```diff
@@ -363,7 +365,7 @@ class CLikeSourceEmitter: public SourceEmitterBase
     void diagnoseUnhandledInst(IRInst* inst);
     void emitInst(IRInst* inst);
 
-    void emitSemantics(IRInst* inst);
+    void emitSemantics(IRInst* inst, bool allowOffsets = false);
```

Comment:
we have a default argument here, would it be clearer to always specify, or are there a lot of cases when we don't care?